patterns/pattern-1/template.yaml

```diff
@@ -550,8 +550,21 @@ Resources:
                     description: Sample JSON format for discovery output
                     default: "{\n  \"document_class\": \"Form-1040\",\n  \"document_description\": \"Brief summary of the document\",\n  \"groups\": [\n    {\n      \"name\": \"PersonalInformation\",\n      \"description\": \"Personal information of Tax payer\",\n      \"attributeType\": \"group\",\n      \"groupType\": \"normal\",\n      \"groupAttributes\": [\n        {\n          \"name\": \"FirstName\",\n          \"dataType\": \"string\",\n          \"description\": \"First Name of Taxpayer\"\n        },\n        {\n          \"name\": \"Age\",\n          \"dataType\": \"number\",\n          \"description\": \"Age of Taxpayer\"\n        }\n      ]\n    }\n  ]\n}"
                     order: 0
+          s3Vectors:
```

Comment:
sectionLabel: S3 Vectors
are these filterable metadata keys tied to use of S3 as the vectorstore?  Seems they should work for any vectorstore (OSS or S3, or others we might add later)..  

Also, maybe they should be removed entirely?  Seems the logic for creating filters in the query is no more in scope for this PR, since I don;t see where these filters would be added to the KB query..